%YAML:1.0
---
cameraMatrix: !!opencv-matrix
   rows: 3
   cols: 3
   dt: d
   data: [ 1.3670257560997277e+03, 0., 9.6635854087126029e+02, 0.,
       1.3683767438047439e+03, 5.5049164789864665e+02, 0., 0., 1. ]
distCoeffs: [ 1.2619525270832671e-01, -1.4678578273216195e-01,
    3.3591463130149398e-03, 1.7833123114949509e-03,
    -3.1631746167135583e-01 ]
